index.html

```diff
@@ -248,12 +248,26 @@ <h3 class="hed-subsection">Picturefill function options</h3>
 				</li>
 			</ul>
 
+			<h3 class="hed-subsection">Picturefill plugins/extensions</h3>
+
+			<p>picturefill core polyfills the core part of responsive images including full support for <code>srcset</code>, <code>sizes</code> and <code>picture</code>/<code>source</code>.</p>
+
+			<p>Some aspects of the native feature are left out to provide a clean and lightweight polyfill.</p>
+
+			<p>In case you need a specific behavior of the native implementation, that is not included in picturefill core, just add the corresponding plugin. The best way is to merge all scripts together.</p>
```

Comment:
It's probaby also good to mention that these can be thrown into a couple <script> tags before pf in case people aren't concatting or don't want to for some reason